synchronization/src/test/java/de/cyface/synchronization/MeasurementSerializerTest.java

```diff
@@ -234,14 +302,14 @@ public void testDecompressDeserialize() throws IOException, DataFormatException,
         final File compressedTransferTempFile = oocut.writeSerializedCompressed(loader, SAMPLE_MEASUREMENT_ID,
                 persistence);
         // Load bytes from compressedTransferFile
-        final byte[] compressedBytes = new byte[(int) compressedTransferTempFile.length()];
+        final byte[] compressedBytes = new byte[(int)compressedTransferTempFile.length()];
         DataInputStream dis = new DataInputStream(new FileInputStream(compressedTransferTempFile));
         dis.readFully(compressedBytes);
 
         // Decompress the compressed bytes and check length and bytes
         Inflater inflater = new Inflater(COMPRESSION_NOWRAP);
         inflater.setInput(compressedBytes, 0, compressedBytes.length);
-        byte[] decompressedBytes = new byte[1000];
+        byte[] decompressedBytes = new byte[2000];
```

Comment:
because now the serialized bytes are ~ 1750 which would be cut when we still use a 1000 Byte array.